doc/undoc-flags.json

```diff
@@ -13,7 +13,7 @@
         "funder-policy",
         "funder-policy-mod",
         "funder-reserve-tank",
-        "lease-fee-base-msat",
+        "lease-fee-base-sat",
```

Comment:
I went back and forth a good bit on what to call this originally. I think this updated version is more explicit as to the actual allowable resolution of the field (it *is* sats); the `msat` designation was left on as a fact that we print out every value as an msat value, for consistency's sake.

Reply:
There was at least one case where there was confusion about `msat` units in [issue 4861](https://github.com/ElementsProject/lightning/issues/4861) Use of `sat`, `msat` and `btc` postfixes should be encouraged.